src/main/java/core/basesyntax/service/CreateReportImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service;
+
+import core.basesyntax.storage.DataStorage;
+import java.util.Map;
+
+public class CreateReportImpl implements CreateReport {
+    private static final String COMA = ",";

```

Comment:
Better make more informative names of constant . Because COMA is a comma anyway))